src/components/molecules/Chatbox/Chatbox.tsx

```diff
@@ -59,29 +74,68 @@ export const Chatbox: React.FC<ChatboxProps> = ({
     [messages, deleteMessage]
   );
 
+  const dropdownOptions = useMemo(
+    () =>
+      OPTIONS.map((option) => (
+        <Dropdown.Item
+          key={option.name}
+          onClick={() => setActiveOption(option)}
+        >
+          {option.name}
+          <FontAwesomeIcon icon={option.icon} />
+        </Dropdown.Item>
+      )),
+    []
+  );
+
+  const showPoll = useMemo(() => activeOption?.name !== OPTIONS[0].name, [
+    activeOption,
+  ]);
+
+  const handleSubmit = useCallback((data) => console.log(data), []);
```

Comment:
`console.log`? Also, can we rename `handleSubmit` something more semantic? What's meant to happen when submit happens?

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1343/commits/f1aa24f7ec8d8d83f24d76aefa459cb1e12080d6